tests/integration/cassandra_auth_test.go

```diff
@@ -0,0 +1,149 @@
+//go:build integration
+// +build integration
+
+/*
+ * Copyright 2022 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package integration
+
+import (
+	"context"
+	"fmt"
+	"github.com/newrelic/nri-cassandra/tests/integration/jsonschema"
+	"github.com/newrelic/nri-cassandra/tests/integration/testutils"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+	"github.com/testcontainers/testcontainers-go"
+	"path/filepath"
+	"strings"
+	"testing"
+	"time"
+)
+
+const (
+	SSLConnectionTimeout = "10000"
+)
+
+type CassandraSSLTestSuite struct {
```

Comment:
just out of curiosity, which are the benefits of using the suites?

To have handy SetupSuite and TearDownSuite?

Reply:
I've added them for setup and teadDown